hack/aks/Makefile

```diff
@@ -43,6 +46,26 @@ azcfg: ## Set the $AZCLI to use aks-preview
 	@$(AZCLI) extension add --name aks-preview --yes
 	@$(AZCLI) extension update --name aks-preview
 
+public-ipv4: rg-up
+	$(AZCLI) network public-ip create --name $(PUBLIC_IPv4) \
+		--resource-group $(GROUP) \
+		--allocation-method Static \
+		--ip-tags $(IP_TAG) \
+		--location $(REGION) \
+		--sku Standard \
+		--tier Regional \
+		--version IPv4
+
+public-ipv6: rg-up
+	$(AZCLI) network public-ip create --name $(PUBLIC_IPv6) \
+		--resource-group $(GROUP) \
+		--allocation-method Static \
+		--ip-tags $(IP_TAG) \
+		--location $(REGION) \
+		--sku Standard \
+		--tier Regional \
+		--version IPv6
```

Comment:
Duplicate `rg-up` calls when creating cluster. I would remove the `rg-up` from the make calls `public-ipv4` and `public-ipv6`

Reply:
updated this